src/store/metadata/stub/store.py

```diff
@@ -65,7 +65,17 @@ def get_edition(self, dataset_id: str, edition_id: str) -> Dict:
         return edition
 
     def get_versions(self, dataset_id: str, edition_id: str) -> Dict:
-        raise NotImplementedError
+        versions_for_edition = [
```

Comment:
so if you look at the setup() method for the stub store we populate self.datasets, self.editions etc. You need to populate a self.versions there as well.

from there (once you've addressed the other comments) you should get a working endpoint.